sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/VersionUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.alibaba.csp.sentinel.util;
```

Comment:
Please add the following license header before the code:

```java
/*
 * Copyright 1999-2018 Alibaba Group Holding Ltd.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Please also add to other new-created classes.